app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -513,11 +505,10 @@ class BrowserTabViewModel(
     private fun updateNetworkLeaderboard(event: TrackingEvent) {
         val networkName = event.trackerNetwork?.name ?: return
         networkLeaderboardDao.incrementNetworkCount(networkName)
-        networkLeaderboardDao.incrementSitesVisited()
```

Comment:
Removed, this is legacy code from before tour use of counts and so no longer makes sense. We only want to increment this when a new page is detected, which we already do elsewhere.